docs/CaliptraIntegrationSpecification.md

```diff
@@ -637,6 +645,11 @@ The following table describes SoC integration requirements.
 | FUSE PAUSER programming rules    | 1 PAUSER attribute register is implemented at SoC interface: CPTRA_FUSE_VALID_PAUSER.                                                                                                                                                     |                          |                                                   |
 | FUSE PAUSER programming rules    | CPTRA_FUSE_PAUSER_LOCK locks the programmable valid pauser register, and marks the programmed value as valid.                                                                                                                             |                          |                                                   |
 | FUSE PAUSER programming rules    | Integrators can choose to harden the valid pauser for fuse access by setting the integration parameter, CPTRA_FUSE_VALID_PAUSER, to the desired value in RTL, and by setting CPTRA_SET_FUSE_PAUSER_INTEG to 1.                            |                          |                                                   |
+| Manufacturing                    | SoC shall provision an IDevID certificate with fields conforming to requirements in [architecture document](https://github.com/chipsalliance/Caliptra/blob/main/doc/Caliptra.md#provisioning-idevid-during-manufacturing). | Statement of conformance | Functionality |
+| Manufacturing                    | Caliptra relies on obfuscation for confidentiality of UDS_SEED. It is strongly advised to implement manufacturing policies to protect UDS_SEED as defense in depth measures -<br>1, Prevent leakage of UDS_SEED on manufacturing floor.<br>2. Implement policies to prevent cloning (programming same UDS_SEED into multiple devices).<br>3. Implement policies to prevent signing of spurious IDEVID certs. | Statement of conformance | Required for Caliptra Threat Model |
+| Chain of Trust                   | SoC shall ensure all mutable code and configuration measurements are stashed into Caliptra. A statement of conformance would list what is considered mutable code and configuration vs what is not. It would also describe the start of the boot sequence of the SoC and how Caliptra is incorporated into it. | Statement of conformance | Required for Caliptra Threat Model | 
+| Chain of Trust                   | SoC shall limit the mutable code and configuration that persists across the Caliptra powergood reset. A statement of conformance would list what does persist and why it is appropriate to do so. | Statement of conformance | Required for Caliptra Threat Model | 
+| Implementation                   | SoC shall apply size only constraints on cells tagged with u__size_only__ string and shall ensure that these are not optimized in synthesis and PNR | Statement of conformance | Required for Caliptra Threat Model                                               |
```

Comment:
`size only constraints`-->`size-only constraints`

`with`-->`with the`

Reply:
Updated.